packages/kit/src/views/Staking/components/UniversalClaim/index.tsx

```
@@ -2,7 +2,9 @@ import type { PropsWithChildren } from 'react';
 import { useCallback, useMemo, useState } from 'react';
 
 import BigNumber from 'bignumber.js';
+import { isNaN } from 'lodash';
```

Biome (1.9.4)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)